.github/workflows/bump_packages.yml

```
@@ -0,0 +1,95 @@
+name: Update packages
+on:
+  workflow_dispatch:
+env:
+  GIT_AUTHOR_NAME: ${{ github.actor }}
+  GIT_AUTHOR_EMAIL: ${{ github.actor }}@users.noreply.github.com
```

I still think we should consider using static values here
```suggestion
  GIT_AUTHOR_NAME: Foreman Packaging Automation
  GIT_AUTHOR_EMAIL: packaging@theforeman.org
```

I'm not sure that email address goes anywhere, but it feels cleaner to indicate it's really automated.